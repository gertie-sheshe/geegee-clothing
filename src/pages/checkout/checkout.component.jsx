import React from 'react';

import './checkout.styles.scss';

const CheckOutPage = () => (
    <div>
        CHECKOUT PAGE
    </div>
)

export default CheckOutPage;
