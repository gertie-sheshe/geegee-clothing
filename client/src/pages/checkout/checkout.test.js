import React from 'react';
import { shallow } from 'enzyme';

import { CheckOutPage } from './checkout.component';

describe('CheckOutPage tests', () => {
  let wrapper;

  beforeEach(() => {
    const mockProps = {
      cartItems: [],
      total: 100,
    };

    wrapper = shallow(<CheckOutPage {...mockProps} />);
  });

  it('Should render Checkout Page', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
