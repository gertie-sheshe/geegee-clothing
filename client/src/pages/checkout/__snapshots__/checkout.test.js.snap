// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CheckOutPage tests Should render Checkout Page 1`] = `ShallowWrapper {}`;
